deploy-manage/remote-clusters/_snippets/eck_rcs_intro.md

```diff
@@ -0,0 +1 @@
+The [remote clusters module](/deploy-manage/remote-clusters.md) in {{es}} enables you to establish uni-directional connections to a remote cluster. This functionality is used in cross-cluster replication (CCR) and cross-cluster search (CCS).
```

Comment:
I don't think we really talk about ES features as modules anymore. I'd consider it to be a dead term

```suggestion
The [remote clusters feature](/deploy-manage/remote-clusters.md) in {{es}} enables you to establish uni-directional connections to a remote cluster. This functionality is used in cross-cluster replication (CCR) and cross-cluster search (CCS).
```